src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -426,8 +426,10 @@ final case class TenantApiServiceImpl(
       certifiedAttribute = targetTenant.attributes.flatMap(_.certified).find(_.id == seed.id)
       updatedTenant <- certifiedAttribute.fold(
         tenantManagementService.addTenantAttribute(targetTenantUuid, seed.toCreateDependency(now))
-      )(_ =>
-        tenantManagementService.updateTenantAttribute(targetTenantUuid, attribute.id, attribute.toCertifiedSeed(now))
+      )(attr => attr.revocationTimestamp match {
+        case None => Future.failed(CertifiedAttributeNotRevoked(targetTenantUuid, attribute.id))
```

Comment:
on the business side the name you used initially was more clear
```suggestion
        case None => Future.failed(CertifiedAttributeAlreadyAssigned(targetTenantUuid, attribute.id))
```